pkg/tsdb/cloudwatch/response_parser_test.go

```diff
@@ -60,7 +61,7 @@ func TestCloudWatchResponseParser(t *testing.T) {
 				Period: 60,
 				Alias:  "{{LoadBalancer}} Expanded",
 			}
-			series, partialData, err := parseGetMetricDataTimeSeries(resp, query)
+			series, partialData, err := parseGetMetricDataTimeSeries(mdrs, labels, query)
```

Comment:
It would be nice to test also that `labels` are having the respected order; therefore I would suggest breaking `parseResponse` in two separate functions.
This would be a safeguard in case the code that building labels would change in the future and would break this behaviour but this only a suggestion.

Reply:
That should be covered through the API tests I'm adding, and things will be changed around radically anyway when the conversion to data frames is over anyway.